packages/dataviews/src/dataform.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import type { FormEventHandler } from 'react'; // TODO: how to import FormEventHandler?
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	TextControl,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { useCallback, useMemo, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { NormalizedField } from './types';
+
+type DataFormProps< Item > = {
+	item: Item;
```

Comment:
In the context of a `form` `item` seems odd 🤔 

Would `values` or something maybe make more sense conceptually?

Reply:
Updated it to `data` at https://github.com/WordPress/gutenberg/pull/63032/commits/1328850a167e48d2c19e69f22df39e652a27d778 to be inline with `<DataViews data={data} />`.